code/deposit-calculator/src/main/java/ru/unn/agile/DepositCalculator/Model/Calculator.java

```diff
@@ -0,0 +1,109 @@
+package ru.unn.agile.DepositCalculator.Model;
+
+public class Calculator {
+
+    private final double minInterestRateForTax = 0.1825;
+    private final double depositTax = 0.35;
+
+    private final int maxPercent = 100;
+    private final int monthsInYear = 12;
+    private final int daysInMonth = 30;
+    private final int daysInYear = 365;
+    private final int daysInQuarter = 90;
+    private final int quarterInYear = 4;
+
+    private double mStartSum;
+    private double mPercent;
+    private double mPeriodCoeff;
+    private int mPeriodInDays;
+    private int mCapitalizationCount;
+    private double mCapitalizationCoeff;
+
+    public Calculator setStartSum(final double startSum) {
+        mStartSum = startSum;
+        return this;
+    }
+
+    public Calculator setPercent(final int percent) {
```

Comment:
Please change parameter type to double

Reply:
This is percent. From 0 to 100. I guess int is much better